src/CollectionsFrameWorkChallenge/HashSetChallenge.java

```diff
@@ -0,0 +1,71 @@
+package CollectionsFrameWorkChallenge;
+
+import java.util.*;
+
+public class HashSetChallenge {
+
+    public static void main(String[] args) {
+        HashSet<String> firstHashSet = new HashSet<>(Arrays.asList("Dog", "Cat", "Tiger", "Kangaroo"));
+        HashSet<String> secondHashSet = new HashSet<>(Arrays.asList("Italy", "France", "United Kingdom"));
+        HashSet<Integer> firstNumberHashSet = new HashSet<>(Arrays.asList(2, 4, 6));
+        HashSet<Integer> secondNumberHashSet = new HashSet<>(Arrays.asList(1, 6));
+        appendASpecifiedElementAtTheEndOfHashSet(firstHashSet);
+        printElementsFromHashSet(firstHashSet);
+        printNumberOfElementsFromHashSet(firstHashSet);
+        emptyHashSet(firstHashSet);
+        isEmpty(firstHashSet); // evaluate return true.
+        HashSet<String> thirdHashSet = new HashSet<>(secondHashSet);
+        convertHashSetToArray(thirdHashSet);
+        convertHashSetToTreeSet(thirdHashSet);
+        convertHashSetToList(thirdHashSet);
+        compareTwoHashSets(secondHashSet, thirdHashSet); // evaluate return true.
+        compareAndPrintCommonElementsFromTwoHashSets(firstNumberHashSet, secondNumberHashSet);
+        removeAllElementsFromAHashSet(thirdHashSet);
```

Comment:
This main method looks much cleaner now, a very good improvement from the past PRs.
Great job!